editor/src/clj/editor/app_view.clj

```diff
@@ -2011,26 +2011,45 @@ If you do not specifically require different script states, consider changing th
       (ui/on-closed! stage (fn [_] (dispose-scene-views! app-view)))
       app-view)))
 
+(defn- make-info-box! []
+  (let [info-panel (HBox.)
+        left-label (Label. "This file is part of a library and cannot be saved.  ")
+        right-link (Hyperlink. "Read more…")
+        spacer (Region.)]
+    (HBox/setHgrow spacer Priority/ALWAYS)
+    (.setStyle info-panel "-fx-background-color: #444; -fx-padding: 4;")
+    (.setTextFill left-label Color/LIGHTGRAY)
+    (.setTextFill right-link Color/CORNFLOWERBLUE)
+    (.setOnAction right-link (ui/event-handler _ (ui/open-url "https://www.defold.com/manuals/editor/")))
```

Comment:
I'll change link when I write docs for it.

Reply:
Here is PR for the manual, I'll replace it in the PR when it will be merged https://github.com/defold/doc/pull/538